### DISCUSSION TITLE
No Index, No Follow - Remove from Index

### DISCUSSION THREAD

**Discussion by Igcorreia** [postalserver/postal#1501] [Feature suggestions]
Please change the application in order it does not index in Serps.
<meta name="robots" content="noindex, noimageindex, nofollow, nosnippet">

**Comment by catphish** [comment]
It should be possible to add some headers to the NGINX proxy config to achieve this.

**Comment by willpower232** [comment]
I believe the quick answer is to add the following to the location blocks in nginx 

```
add_header X-Robots-Tag "noindex, nofollow, nosnippet, noarchive";
```

but it might be more useful to add in the meta tag to the html and haml.

**Comment by Igcorreia** [comment]
To make sure nothing is indexed I added to all path/pages/assets:

UPDATED to remove extra space as per @willpower232  comment.

    location / {
        add_header X-Robots-Tag "noindex, nofollow, nosnippet, noarchive";
        ....
    }

    location /assets {
        add_header X-Robots-Tag "noindex, nofollow, nosnippet, noarchive";
        ....
    }

    location @puma {
        add_header X-Robots-Tag "noindex, nofollow, nosnippet, noarchive";
        ....
    }

**Comment by willpower232** [comment]
@Igcorreia I've just spotted that there is an extra space between `add_header` and `X-Robots-Tag` so you should remove that so the header is correct and not ` X-Robots-Tag`. I'll update my original comment.
